server/views/components/work-media/work-media.njk

```diff
@@ -0,0 +1,13 @@
+<div class="row bg-black work-media js-work-media">
+  <div class="work-media__controls js-work-media-controls pointer-events-none">
+    {% if model.queryString %}
+      <div class="flush-grid-left {{ {s:2} | spacingClasses({padding: ['top']}) }}">
+        {% componentV2 'return-to-results', {queryString: model.queryString}, null, {extraClasses: ['js-work-media-control', 'pointer-events-all']} %}
```

Comment:
Yeah, you're probably right – I think I wanted to avoid writing any component-specific styles, but as a 'return-to-results' component, it's already not really abstract enough to be much use.